cartesian_impedance_msgs/msg/NullSpace.msg

```diff
@@ -0,0 +1,11 @@
+# parameters for a redundant joint of the robot when in Cartesian Impedance
+# the values are divided into translational and rotational
```

Comment:
Does this mean all redundant joints will have the same damping/stiffness?  Is this typical?  Could we use a `joint_names` and float array instead?